go.mod

```diff
@@ -0,0 +1,75 @@
+module github.com/intel/pmem-csi
+
+go 1.13
+
+require (
+	github.com/Microsoft/go-winio v0.4.14 // indirect
+	github.com/beorn7/perks v1.0.1 // indirect
+	github.com/container-storage-interface/spec v1.2.0
+	github.com/docker/go-units v0.4.0
+	github.com/docker/spdystream v0.0.0-20181023171402-6480d4af844c // indirect
+	github.com/evanphx/json-patch v4.5.0+incompatible // indirect
+	github.com/gogo/protobuf v1.3.1 // indirect
+	github.com/golang/groupcache v0.0.0-20191027212112-611e8accdfc9 // indirect
+	github.com/golang/protobuf v1.3.2
+	github.com/google/go-cmp v0.3.1 // indirect
+	github.com/google/uuid v1.1.1
+	github.com/googleapis/gnostic v0.3.1 // indirect
+	github.com/hashicorp/golang-lru v0.5.3 // indirect
+	github.com/imdario/mergo v0.3.8 // indirect
+	github.com/json-iterator/go v1.1.8-0.20191012130704-03217c3e9766 // indirect
+	github.com/kubernetes-csi/csi-lib-utils v0.6.1
+	github.com/kubernetes-csi/csi-test v1.1.2-0.20191016154743-6931aedb3df0
+	github.com/onsi/ginkgo v1.10.2
+	github.com/onsi/gomega v1.7.0
+	github.com/pkg/errors v0.8.1
+	github.com/prometheus/client_model v0.0.0-20190812154241-14fe0d1b01d4 // indirect
+	github.com/prometheus/common v0.4.1 // indirect
+	github.com/prometheus/procfs v0.0.5 // indirect
+	github.com/spf13/pflag v1.0.5 // indirect
+	github.com/stretchr/testify v1.4.0 // indirect
+	go.uber.org/atomic v1.4.0 // indirect
+	go.uber.org/multierr v1.2.0 // indirect
+	go.uber.org/zap v1.11.0 // indirect
+	golang.org/x/crypto v0.0.0-20191011191535-87dc89f01550 // indirect
+	golang.org/x/net v0.0.0-20191027233614-53de4c7853b5
+	golang.org/x/sys v0.0.0-20191105231009-c1f44814a5cd // indirect
+	golang.org/x/time v0.0.0-20191024005414-555d28b269f0 // indirect
+	google.golang.org/appengine v1.6.5 // indirect
+	google.golang.org/genproto v0.0.0-20191009194640-548a555dbc03 // indirect
+	google.golang.org/grpc v1.24.0
+	gopkg.in/inf.v0 v0.9.1 // indirect
+	gopkg.in/square/go-jose.v2 v2.4.0 // indirect
+	k8s.io/api v0.0.0
+	k8s.io/apimachinery v0.0.0
+	k8s.io/client-go v0.0.0
+	k8s.io/component-base v0.0.0
+	k8s.io/klog v1.0.0
+	k8s.io/kube-openapi v0.0.0-20190918143330-0270cf2f1c1d // indirect
+	k8s.io/kubernetes v1.17.0-alpha.0.0.20191109043140-c3f2a6524ed8
+	k8s.io/utils v0.0.0-20190801114015-581e00157fb1
+)
+
+replace (
+	k8s.io/api => k8s.io/api v0.0.0-20191016110408-35e52d86657a
```

Comment:
How are we going to maintain this section in the future?

Two options:
- use a script like the one I wrote for kubernetes-csi: https://github.com/kubernetes-csi/csi-release-tools/blob/master/go-get-kubernetes.sh
- wait for Kubernetes 1.17, which will have proper tags that'll show up here instead of "v0.0.0-20191016110408-35e52d86657a"

Reply:
BTW, how was this section created?

Reply:
> BTW, how was this section created?

Using the same release script you pointed.

Reply:
I was in the impression that we might not need to do more update rounds before going to Kubernetes v1.17

Reply:
Okay, so let's just keep in mind that this script was used and if one of us three developers needs to change the Kubernetes code we'll know what to do.